cmd/main.go

```diff
@@ -6,6 +6,7 @@ package main
 import (
 	"flag"
 	"os"
+	"sigs.k8s.io/cli-utils/cmd/destroy"
```

Comment:
This import should not be in this section.

Reply:
Will do